code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -33,13 +33,19 @@
 		if(!fullpower)
 			src << "<span class='warning'>Your powers are useless on this holy ground.</span>"
 			return 0
+	if(check_holy())
+		if(!fullpower)
+			var/turf/simulated/T = get_turf(src)
+			if(T.lighting_lumcount > 2)
```

Comment:
This threshold (```2```) appears multiple times, make it a define (or var, or config option, etc).
Rule of thumb is that any literal (bare data value) that appears more than once should be referred to with an identifier
Better for clarity, maintainability, etc